src/index.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          class="logo" 
+          src="images/logo.png" 
+          alt="logo">
```

Comment:
the closing tag should be lined with opened tag
```suggestion
        <img 
          class="logo" 
          src="images/logo.png" 
          alt="logo"
        >
```